recipes/gmp/gmp_6.1.1.oe

```diff
@@ -1,4 +1,4 @@
 require ${PN}.inc
 LICENSE = "LGPL-3.0+"
 
-S = "${SRCDIR}/${PN}-6.0.0"
+S = "${SRCDIR}/${PN}-6.1.1"
```

Comment:
Please just remove this line. It was only added for that one-off strange 6.0.0a release.